prometheus_remediation/job_runner.py

```diff
@@ -136,13 +144,29 @@ def run_job_from_alert(event: PrometheusKubernetesAlert, params: JobParams):
 
     job.create()
 
+
+    info_messages = []
     if params.notify:
-        event.add_enrichment([MarkdownBlock(f"Created job from alert: *{job_name}*.")])
+        info_messages.append(f"Created Job from alert: *{job_name}*.")
```

Comment:
Can you make the "Create Job from alert" bold and the job itself not bold. (That will better match the style in the rest of the Slack message.)